### PR TITLE
Added isAlpha method

### PR DESCRIPTION
**Pull request by RobFairclough**
Added the isAlpha function, fixed an export typo from last PR and should pass all CI this time 😅

### CODE REVIEW

**Comment by vorillaz** [comment]
Hey, @RobFairclough can you please rebase your local branch and we are ready to merge!

**Comment by RobFairclough** [comment]
All done, first time using rebase so please let me know if there's any issues!

File: lerna.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.0.12",
```
Comment (vorillaz) [inline lerna.json]: Just use version 0.0.12, during the release phase lerna will handle upgrades.

File: packages/isAlpha/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@plexis/is-alpha",
+  "version": "0.0.13",
```
Comment (vorillaz) [inline packages/isAlpha/package.json]: Ditto.

File: packages/plexis/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "plexis",
-  "version": "0.0.12",
+  "version": "0.0.13",
```
Comment (vorillaz) [inline packages/plexis/package.json]: Ditto.

File: packages/plexis/package.json [hunk 4]
```diff
@@ -9,11 +9,12 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@plexis/to-lower": "^0.0.12",
-    "@plexis/to-pred": "^0.0.12",
-    "@plexis/to-succ": "^0.0.12",
-    "@plexis/to-title": "^0.0.12",
-    "@plexis/without-diacritics": "^0.0.12",
+    "@plexis/to-lower": "^0.0.13",
```
Comment (vorillaz) [inline packages/plexis/package.json]: Same here, you shall add the  `^0.0.12` version

**Comment by vorillaz** [comment]
Everything seems good @RobFairclough thanks for your contribution.

**Pull request merged**